package.json

```diff
@@ -4,9 +4,11 @@
   "description": "A UCOSP project for building a Z-Index Devtool",
   "main": "index.js",
   "scripts": {
-    "start": "budo src --serve bundle.js --dir docs",
-    "build": "browserify src > docs/bundle.js",
-    "test": "mocha --compilers js:babel-register test/stacking-context-actions.js"
+    "build": "NODE_ENV=development webpack --watch",
+    "start-app": "http-serve ./docs",
+    "start-ext": "web-ext run --source-dir ./extension",
```

Comment:
I don't know if the `run` command also generates a zip file to a directory like the `build` command does, but if it does, it will be needed to also add that directory to the .gitignore file.